lighting_advanced_search/wizard/product_advanced_search.py

```diff
@@ -100,12 +100,8 @@ class LightingProductAdvancedSearch(models.TransientModel):
                                                 column1='advanced_search_id',
                                                 column2='color_temperature_id')
     # Luminous flux search
-    flux_in_ids = fields.Many2many(string='Luminous flux',
-                                   comodel_name='lighting.product.flux',
-                                   relation='lighting_product_advanced_search_flux_in_rel',
-                                   column1='advanced_search_id',
-                                   column2='flux_id')
-
+    flux_from_in = fields.Float(string='From')
+    flux_to_in = fields.Float(string='To')
```

Comment:
Good approach!!
Just rename according to the new field name `nominal_flux_from_in` and `nominal_flux_to_in`